### DISCUSSION TITLE
unique_id column type

### DISCUSSION THREAD

**Discussion by hadoan88** [moj-analytical-services/splink#2270] [General]
Can I set the UUID value for unique_id column instead of numeric?

**Comment by RobinL** [comment]
It can def be string or numeric.  If I remember correctly the data type of the unique id column can actually be anything.  Are you getting an error?

**Reply by hadoan88** [reply]
Thanks @RobinL  , I will try on my end

**Comment by vfrank66** [comment]
I use a UUID for unique_id column and it works just fine on splink=3.9.14/3.9.15

**Reply by hadoan88** [reply]
Thanks @vfrank66 , I will try on my end
